lib/features/game/views/page_view_pages/question_page.dart

```diff
@@ -22,12 +22,15 @@ class QuestionPage extends StatelessWidget {
               padding: const EdgeInsets.symmetric(horizontal: 32),
               child: orientation == Orientation.portrait
                   ? Column(children: [
+                      SizedBox(height: 20),
```

Comment:
If you need padding on whole column add it above column using padding. In this case padding is already there so just add appropriate amount

Reply:
done